internal/pkg/cli/pipeline_init.go

```diff
@@ -274,6 +302,35 @@ func (o *initPipelineOpts) createBuildspec() (string, error) {
 	return path, nil
 }
 
+func (o *initPipelineOpts) getArtifactBucket() ([]artifactBucket, error) {
+	proj, err := o.storeSvc.GetProject(o.ProjectName())
+	if err != nil {
+		return nil, fmt.Errorf("get project %s: %w", o.ProjectName(), err)
```

Comment:
nit: I've been adding "metadata" whenever referring to errors from SSM, what do you think of:
```suggestion
		return nil, fmt.Errorf("get project metadata %s: %w", o.ProjectName(), err)
```

Reply:
I agree. And we should be consistent.